src/check/parse/IR.zig

```diff
@@ -1775,18 +1775,18 @@ pub const NodeStore = struct {
         statements: []const StatementIdx,
         region: Region,
 
-        pub fn toSExpr(self: @This(), gpa: Allocator, env: *base.ModuleEnv, ir: *IR) anyerror!sexpr.Node {
-            var file_node = try sexpr.Node.newNode(gpa, "file");
+        pub fn toSExpr(self: @This(), gpa: Allocator, env: *base.ModuleEnv, ir: *IR) anyerror!sexpr.Expr {
```

Comment:
I think you can remove the `anyerror!` from the `toSExpr` functions and that will lead to removing the rest of the `try` statements? Though maybe I am missing another form of error.

Reply:
Done in abd03a400b